### ISSUE TITLE
Grafana "Origin Not Allowed" 403 Error

### ISSUE THREAD

**Issue by vijaybiradar** [prometheus-community/helm-charts#2683]
### Describe the bug a clear and concise description of what the bug is.

What happened: Receiving an "Origin Not Allowed" 403 error when POST or PUT requests are made from the browser

installed grafana with kube-prometheus-stack.

Environment:
kube-prometheus-stack-grafana
Grafana version: (grafana:9.2.3)

Anything workaround we need to add to fix this error

![image](https://user-images.githubusercontent.com/38376802/201888354-e619cbc4-ce91-4761-81b5-00247b0eccb3.png)

### What's your helm version?

Version:v3.7.2

### What's your kubectl version?

Major:"1", Minor:"23", GitVersion:"v1.23.5"

### Which chart?

*

### What's the chart version?

*

### Enter the command that you execute and failing/misfunctioning.

kube-prometheus-stack   kube-prom       3               2022-11-07 07:53:27.411359635 +0000 UTC deployed        kube-prometheus-stack-41.7.0        0.60.1

**Comment by serhii-satanenko** [comment]
here is what i did:
1. I deployed kube-prometheus-stack chart in my cluster with nginx-ingress installed

2. Then I saw that my login to grafana and prometheus was not working.

3. I found a solution to this problem at https://github.com/prometheus-community/helm-charts/issues/830

4. Then I destroyed this table, (as written in the instructions):
https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack


5. And then changed the value as follows:
Exit:
        included: true
        annotations:
          kubernetes.io/ingress.class: nginx
          nginx.ingress.kubernetes.io/service-upstream: "true"
          nginx.ingress.kubernetes.io/upstream-vhost: "kube-prometheus-stack-prometheus.monitoring.svc.cluster.local"

6. After that I expanded the chart again and got this error all the time. For example, when I try to change the admin password
